src/Controller/SlackController.php

```diff
@@ -20,13 +22,15 @@ public function index(): Response
     }
 
     #[Route('/createchannel', name: 'create_channel')]
-    public function createChannel(SlackService $slackService): Response
+    public function createChannel(SlackService $slackService, Idea $ideaTitle): Response
     {
-        $channelName = 'testnbidk';
-        // Set the channel name to 'channelname' (replace with desired channel name)
-        // Set in slug cf doc symfony -> automatically create a channel from an idea
-        $channel = $slackService->createChannel($channelName);
-        // Call the createChannel method of SlackService with the specified channel name
+        $channelName = $ideaTitle->getTitle(); // Set the channel name based on idea name
+
+        $slugger = new AsciiSlugger(); // Create a new instance of the AsciiSlugger
```

Comment:
No ! Use a SluggerInterface as showed in the documentation : https://symfony.com/doc/current/components/string.html#slugger